sourcegraph/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "integration_id": "sourcegraph-services-map",
+  "categories": [
+    "monitoring",
+    "collaboration",
+    "source control"
+  ],
+  "app_uuid": "19916057-6b27-4c17-89c5-821794532315",
+  "creates_events": false,
+  "display_name": "Sourcegraph Datadog Services Map Extension",
+  "manifest_version": "2.0.0",
+  "guid": "b647480e-bdfb-4cf6-baa2-e773f69ac466",
```

Comment:
As nick mentioned, we need to fix this file :)

Reply:
Fixed!